crates/core/c8y_smartrest/src/smartrest_deserializer.rs

```diff
@@ -242,6 +242,28 @@ impl SmartRestRestartRequest {
     }
 }
 
+#[derive(Debug, Deserialize, Serialize, PartialEq)]
+pub struct SmartRestConfigUploadRequest {
+    pub message_id: String,
```

Comment:
You did like this in your alarm implementation :P
```suggestion
    pub message_id: i32,
```